quickwit-common/src/net.rs

```diff
@@ -116,6 +200,42 @@ pub fn find_available_tcp_port() -> anyhow::Result<u16> {
     Ok(port)
 }
 
+/// Attempts to find the private IP of the host. Returns the matching interface name along with it.
+pub fn find_private_ip() -> Option<(String, IpAddr)> {
+    _find_private_ip(datalink::interfaces())
+}
+
+fn _find_private_ip(interfaces: Vec<NetworkInterface>) -> Option<(String, IpAddr)> {
```

Comment:
nitpick: does not matter here, but really does in some other place to the point where it should be a clippy lint imho.

```suggestion
fn _find_private_ip(interfaces: &[NetworkInterface]) -> Option<(String, IpAddr)> {
```

Reply:
I did not fix it on the caller sites.

Reply:
Yeah, makes sense, I was lazy.